content/getting-started/quickstart.textile

```diff
@@ -510,12 +526,8 @@ channel.Publish("greeting", "hello!");
 ```
 
 ```[python] 
-async def main():
-  ably = AblyRest('{{API_KEY}}')
-  channel = ably.channels.get('quickstart')
-  await channel.publish(u'greeting', u'hello!')
-  # other code such as publish, close
-asyncio.run(main())
+channel = ably.channels.get('quickstart')
+await channel.publish(u'greeting', u'hello!')
```

Comment:
unicode strings are the default in python 3.0 (ie all currently supported versions of python)
```suggestion
await channel.publish('greeting', 'hello!')
```

Reply:
Good to know 👍 Updated in [6d867e2](https://github.com/ably/docs/pull/1944/commits/6d867e2337199854e6dd60b46e944b748d96f52c)